bin/postgres-ha/bootstrap-postgres-ha.sh

```diff
@@ -168,32 +168,33 @@ then
         > "${PATRONI_POSTGRESQL_DATA_DIR}/postgresql.conf"
     fi
 
-    manual_start_pg_ctl_options=""
-    # detect if this is a server that needs to entre recovery mode, which is
-    # what might happen after an instance is cloned.
-    # if this is the case, we will want to have a warm standby
+    manual_start_pg_ctl_options=()
     if [[ -f "${PATRONI_POSTGRESQL_DATA_DIR}/recovery.conf" || -f "${PATRONI_POSTGRESQL_DATA_DIR}/recovery.signal" ]]
     then
-      echo_info "Discovered presence of a recovery.conf or recovery.signal file"
-      echo_info "Setting hot_standby=\"off\" until PostgreSQL reaches a consistent state"
```

Comment:
Why remove these?

Reply:
The second line isn't accurate. We're setting more than that one parameter now, and they stay in place until the restart around L75.

The first line should probably stay. When PostgreSQL starts it also logs that it is going into recovery, but if it _doesn't_ start this is the only indication we mucked with things.

Reply:
OK.